djangocms_navigation/cms_menus.py

```diff
@@ -48,9 +49,26 @@ def get_roots(self, request):
             if hasattr(request, "toolbar") and request.toolbar.edit_mode_active:
                 inner_filter["versions__state__in"] += [DRAFT]
             menucontents = versionable.distinct_groupers(**inner_filter)
+
+            if getattr(settings, "MAIN_NAVIGATION_ENABLED", False):
+                menucontents = self.get_main_navigation(menucontents=menucontents)
+
             queryset = queryset.filter(menucontent__in=menucontents)
         return queryset
 
+    def get_main_navigation(self, menucontents):
+        """
+        Takes a queryset of MenuContent objects, filters to include only those where the related Menu object has been
```

Comment:
Would you be able to follow this format for params/returns in this docstring?
https://github.com/FidelityInternational/djangocms-navigation/blob/f10af485558b8dbc6d199cefa5846bd7a5aece57/djangocms_navigation/helpers.py#L14

Although this hasn't been incorporated throughout the package already, we should try to ensure any changes we make are pushing the highest quality possible. This isn't something we have discussed though, so I wouldn't have expected it to be picked up implicitly!